drivers/hmis/app/models/hmis/hud/validators/enrollment_validator.rb

```diff
@@ -110,7 +110,8 @@ def self.hmis_validate(record, role: nil, **_)
 
   def validate(record)
     super(record) do
-      record.errors.add :project_id, :required if record.project_id.nil? && record.wip.nil?
+      # enrollment.project_id should match the project.project_id. The rails association is through actual_project_id
+      record.errors.add :project_id, 'must match project pk' if record.ProjectID && record.ProjectID != record.project.ProjectID
```

Comment:
Is the error string correct?  I feel like the error is catching something different.
```
'must match project pk'
```